search-endpoint.md

```diff
@@ -169,6 +165,17 @@ The returned JSON response will be like:
     "facets" : [
       {
         "name" : "author",
+        "facetType": "text",
+        "hasMore": true,
```

Comment:
you don't need that. If there are more author facet values you will have a next link

Reply:
Correct. I'll check if we can remove this field.